packages/web-components/src/components/va-number-input/va-number-input.tsx

```diff
@@ -46,11 +46,6 @@ export class VaNumberInput {
    */
   @Prop() required?: boolean = false;
 
-  /**
-   * The inputmode attribute.
-   */
-  @Prop() inputmode?: 'decimal' | 'numeric';
```

Comment:
My only concern was that removing the decimal/numeric would remove some sort of validation we were relying on, but `inputmode` only affects what [keyboard is displayed on digital](https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/). So 👍

Reply:
I think that's a good callout because I see a lot of instances of this prop being used in vets-website. Besides leaving behind some cruft, do we think this should be considered a breaking `major` change?

![Screenshot 2023-08-18 at 2 25 51 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/fb3ed9ce-4a65-4ad8-a6a6-f2b80ce0d57c)